tests/components/alarm_control_panel/test_device_action.py

```diff
@@ -297,6 +320,13 @@ async def test_action(hass):
         == STATE_ALARM_ARMED_NIGHT
     )
 
+    hass.bus.async_fire("test_event_arm_vacation")
+    await hass.async_block_till_done()
+    assert (
+        hass.states.get("alarm_control_panel.alarm_no_arm_code").state
+        != STATE_ALARM_ARMED_VACATION
```

Comment:
What is the goal of this test?

Reply:
This was added because with "==" build fails everytime. Please provide a solution if you know how to solve

Reply:
this one keeps giving troubles with using "==". See https://github.com/home-assistant/core/actions/runs/661327554

Reply:
tried everything, this is not related to wrong coding. i give up.

Reply:
@posixx ill take a look and see if I can help with tests issue.